### PR TITLE
Jv ticket 11

### LINKED ISSUE #11: Increase maximum possible price
The system currently will reject any product being created with a price greater than $10,000. Increase that maximum to $17,500.

### PR DESCRIPTION
**Pull request by JeremiahV93**
Increases Product maximum price to $17,500.00
## Changes

- Increases Product maximum price to $17,500.00

## Testing

Description of how to test code...

- [ ] Create product with higher than $17,500.00

## Related Issues

- Fixes #11

### CODE REVIEW

**Review by hannahhall — COMMENTED** [review]
Code looks good but it has some other commits for different tickets. Make sure you branch off of master when working on something new. I'll approve once the other ticket is merged

**Comment by JeremiahV93** [comment]
Merged  with changes from previous branch

**Review by hannahhall — APPROVED** [review]
👍 approved

**Pull request merged**
